Source/carthage/Update.swift

```diff
@@ -32,11 +32,11 @@ public struct UpdateCommand: CommandType {
 public struct UpdateOptions: OptionsType {
 	public let buildAfterUpdate: Bool
 	public let configuration: String
-	private let checkoutOptions: CheckoutOptions
+	internal let checkoutOptions: CheckoutOptions
```

Comment:
[Bootstrap evaluates UpdateOptions.](https://github.com/jdhealy/Carthage/blob/22d5892fa9a485aa2d0c68e1bf879ba439608260/Source/carthage/Bootstrap.swift#L23)

[We use the colorOptions here.](https://github.com/jdhealy/Carthage/blob/22d5892fa9a485aa2d0c68e1bf879ba439608260/Source/carthage/Bootstrap.swift#L31)